src/main/java/net/phptravels/gui/login/HomePage.java

```diff
@@ -17,84 +15,216 @@
+    
+    @FindBy(xpath = "//form[@id = 'hotels-search']//select[@id = 'nationality']")
+    private ExtendedWebElement hotelNationalitySelect;
     
-    @FindBy(xpath = "//button[text()= ' flights']")
+    @FindBy(xpath = "//form[@id = 'hotels-search']//button[@type = 'submit']")
+    private ExtendedWebElement hotelSearchButton;
+
+    @FindBy(xpath = "//div//li[@data-position =2]")
     private ExtendedWebElement flightButton;
     
-    @FindBy(id = "tours-tab")
-    private ExtendedWebElement toursButton;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@id = 'one-way']")
+    private ExtendedWebElement flightsOneWayRadiobutton;
     
-    @FindBy(id = "visa-tab")
-    private ExtendedWebElement visaButton;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@id = 'round-trip']")
+    private ExtendedWebElement flightsRoundTripRadioButton;
+    
+    @FindBy( xpath = "//div[@id = 'flights']//select[@id = 'flight_type']")
+    private ExtendedWebElement flightsTypeDropdown;
     
-    @FindBy(id = "one-way")
-    private ExtendedWebElement oneWayRadiobutton;
+    @FindBy( xpath = "//input[@id = 'autocomplete']")
+    private ExtendedWebElement flightsFlyingFromCombobox;
     
-    @FindBy(id = "round-trip")
-    private ExtendedWebElement roundTripRadioButton;
+    @FindBy( xpath = "//input[@id = 'autocomplete2']")
+    private ExtendedWebElement flightsFlyingToCombobox;
     
-    @FindBy(id = "flight_type")
-    private ExtendedWebElement flightTypeDropdown;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@class = 'depart form-control']")
+    private ExtendedWebElement flightsDepartureDate;
     
-    @FindBy(id = "autocomplete")
-    private ExtendedWebElement flyingFromCombobox;
+    @FindBy(xpath ="//div[@id = 'flights']//a[@class = 'dropdown-toggle dropdown-btn waves-effect']")
+    private ExtendedWebElement flightsPassengersDropdown;
     
-    @FindBy(id = "autocomplete2")
-    private ExtendedWebElement flyingToCombobox;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@id = 'fadults']")
+    private ExtendedWebElement flightsAdults;
     
-    @FindBy(name = "depart")
-    private ExtendedWebElement departureDate;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@id = 'fchilds']")
+    private ExtendedWebElement flightsChilds;
     
-    @FindBy(xpath ="//a//span[text()= '1']")
-    private ExtendedWebElement passengersDropdown;
+    @FindBy(xpath = "//div[@id = 'flights']//input[@id = 'finfant']")
+    private ExtendedWebElement flightsInfants;
     
-    @FindBy(id = "flights-search")
-    private ExtendedWebElement searchFlightButton;
+    @FindBy( xpath = "//div[@class='autocomplete-wrapper _1 row_1']//div[@class = 'autocomplete-location']")
+    private ExtendedWebElement flightsFlyingFromSearch;
+    
+    @FindBy( xpath = "//div[@class='autocomplete-wrapper _1 row_2']//div[@class = 'autocomplete-location']")
+    private ExtendedWebElement flightsFlyingToSearch;
```

Comment:
as I understand it is the elements from the screenshot. I suggest it to rename to flightsFlyingFromSearch**Result**
<img width="326" alt="image" src="https://user-images.githubusercontent.com/103670717/163992532-7613a0ea-122e-43b0-a865-d7ed36fdfe08.png">